redi-connect-front/src/components/organisms/RediHero.scss

```diff
@@ -0,0 +1,37 @@
+@import "~bulma/sass/utilities/_all";
+@import '../../styles/variables.scss';
+
+.hero {
+  &__column {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+
+    &--heading.title {
+      line-height: 4.5rem;
+      margin-bottom: 1rem;
+
+      @include mobile() {
+        font-size: 3rem;
+        line-height: 3.125rem;
+      }
+    }
+
+    &--content {
+      font-size: $size-4;
+      line-height: 2rem;
+      margin-bottom: 3rem;
+
+      @include mobile() {
+        margin-bottom: .75rem;
+        font-size: $size-5;
+        line-height: 1.5rem;
+      }
+    }
+
+    &--img {
+      float: right;
```

Comment:
what is this float for? is this really needed in the colums setup?

Reply:
True, has been removed